test/unit/runner.spec.js

```diff
@@ -119,9 +125,12 @@ describe('Runner', function () {
     });
 
     it('should not fail when a new common global is introduced', function () {
+      if (process.browser) {
```

Comment:
though I think the lines [here](https://github.com/mochajs/mocha/pull/2868/files#diff-84453848e82796cadaea28db8fca1ab2R84) might mean this never happens.